src/index.html

```diff
@@ -1,27 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Should we keep an index page with some info on how to get to the demos? Otherwise, there's no indication on what to do next (I know we know, but it'd be nice for this to be clear for anyone testing the repo out):

<img width="537" alt="Screen Shot 2019-12-18 at 4 16 10 PM" src="https://user-images.githubusercontent.com/197440/71134094-d8e47400-21b1-11ea-9135-ac571a474dc2.png">

Reply:
Yep! will add.